modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -7,7 +7,27 @@
 
 The default installation of metering configures Hive to use an embedded Java database called Derby. This is unsuited for larger environments and can be replaced with either a MySQL or PostgreSQL database. Use the following example configuration files if your deployment requires a MySQL or PostgreSQL database for Hive.
 
-There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
+There are three configuration options you can use to control the database used by Hive metastore: url, driver and secretName.
+
+
+You will create your MySQL or Postgres instance with a username and password. Then either in the OpenShift CLI or by using a yaml file, you will make a secret. The `secretName` you use for this secret needs to map to your MeteringConfig `spec.hive.spec.config.db.secretName`.
```

Comment:
s/You will create your MySQL/You can create a MySql or Postgres instance with a username and password. Create a secret through either the CLI or in a yaml file.